migrate/20210623152412_create_tables.go

```diff
@@ -0,0 +1,366 @@
+			CREATE TABLE metadata_submit_posts (
+				transaction_hash    BYTEA PRIMARY KEY,
+				post_hash_to_modify BYTEA  NOT NULL,
+				parent_stake_id     BYTEA  NOT NULL,
+				body                BYTEA  NOT NULL,
+				timestamp_nanos     BIGINT NOT NULL,
+				is_hidden           BOOL   NOT NULL
+			);
+		`)
+		if err != nil {
+			return err
+		}
+
+		_, err = db.Exec(`
+			CREATE TABLE metadata_update_exchange_rates (
+				transaction_hash      BYTEA PRIMARY KEY,
+				usd_cents_per_bitcoin BIGINT NOT NULL
+			);
+		`)
+		if err != nil {
+			return err
+		}
+
+		_, err = db.Exec(`
+			CREATE TABLE metadata_update_profiles (
+				transaction_hash         BYTEA PRIMARY KEY,
+				profile_public_key       BYTEA,
+				new_username             BYTEA,
+				new_description          BYTEA,
+				new_profile_pic          BYTEA,
+				new_creator_basis_points BIGINT NOT NULL
+			);
+		`)
+		if err != nil {
+			return err
+		}
+
+		_, err = db.Exec(`
+			CREATE TABLE metadata_follows (
+				transaction_hash    BYTEA PRIMARY KEY,
+				followed_public_key BYTEA NOT NULL,
+				is_unfollow         BOOL NOT NULL
+			);
+		`)
+		if err != nil {
+			return err
+		}
+
+		_, err = db.Exec(`
+			CREATE TABLE metadata_likes (
+				transaction_hash BYTEA PRIMARY KEY,
+				liked_post_hash  BYTEA NOT NULL,
+				is_unlike        BOOL NOT NULL
+			);
+		`)
+		if err != nil {
+			return err
+		}
+
+		_, err = db.Exec(`
+			CREATE TABLE metadata_creator_coins (
+				transaction_hash                BYTEA PRIMARY KEY,
+				profile_public_key              BYTEA NOT NULL,
+				operation_type                  SMALLINT NOT NULL,
+				bit_clout_to_sell_nanos         BIGINT NOT NULL,
+				creator_coin_to_sell_nanos      BIGINT NOT NULL,
+				bit_clout_to_add_nanos          BIGINT NOT NULL,
+				min_bit_clout_expected_nanos    BIGINT NOT NULL,
+				min_creator_coin_expected_nanos BIGINT NOT NULL
+			);
+		`)
+		if err != nil {
+			return err
+		}
+
+		_, err = db.Exec(`
+			CREATE TABLE metadata_creator_coin_transfers (
+				transaction_hash               BYTEA PRIMARY KEY,
+				profile_public_key             BYTEA NOT NULL,
+				creator_coin_to_transfer_nanos BIGINT NOT NULL,
+				receiver_public_key            BYTEA NOT NULL
+			);
+		`)
+		if err != nil {
+			return err
+		}
+
+		_, err = db.Exec(`
+			CREATE TABLE metadata_swap_identities (
+				transaction_hash BYTEA PRIMARY KEY,
+				from_public_key  BYTEA NOT NULL,
+				to_public_key    BYTEA NOT NULL
+			);
+		`)
+		if err != nil {
+			return err
+		}
+
+		_, err = db.Exec(`
+			CREATE TABLE notifications (
+				transaction_hash BYTEA PRIMARY KEY,
+				mined            BOOL NOT NULL,
+				to_user          BYTEA NOT NULL,
+				from_user        BYTEA NOT NULL,
+				other_user       BYTEA,
+				type             SMALLINT NOT NULL,
+				amount           BIGINT,
+				post_hash        BYTEA,
+				timestamp        BIGINT NOT NULL
+			);
+		`)
+		if err != nil {
+			return err
+		}
+
+		_, err = db.Exec(`
+			CREATE TABLE profiles (
```

Comment:
It would be great if we could turn off pluralization so that it is easier to find the struct corresponding to a specific postgres table.  For example, this table would be "profile" since that maps directly to the "profile" struct.

Reply:
I resolved this by manually defining all of the table names in the corresponding structs